### PR TITLE
add struct AuditLogDiff

### PR DESCRIPTION
**Pull request by wardab0104**
add struct AuditLogDiff

### CODE REVIEW

**Comment by wardab0104** [comment]
@marselab @angelo-chan please help review

File: eventstream.go [hunk 1]
```diff
@@ -416,7 +416,12 @@ func NewAuditLogBuilder() *AuditLogBuilder {
 
 type AuditLogPayload struct {
 	Content map[string]interface{} `json:"content"`
-	Diff    map[string]interface{} `json:"diff"`
+	Diff    AuditLogDiff           `json:"diff"`
+}
+
+type AuditLogDiff struct {
+	Before map[string]interface{} `json:"before,omitempty"`
+	After  map[string]interface{} `json:"after,omitempty"`
```
Comment (Allen8975701) [inline eventstream.go:424]: if diff exists, then before & after should be required

Comment (wardab0104) [reply eventstream.go:424]: updated

File: eventstream.go [hunk 2]
```diff
@@ -416,7 +416,12 @@ func NewAuditLogBuilder() *AuditLogBuilder {
 
 type AuditLogPayload struct {
 	Content map[string]interface{} `json:"content"`
-	Diff    map[string]interface{} `json:"diff"`
+	Diff    AuditLogDiff           `json:"diff"`
+}
+
+type AuditLogDiff struct {
+	Before map[string]interface{} `json:"before,omitempty"` // required if AuditLogDiff is not nil
```
Comment (angelo-chan) [inline eventstream.go:423]: this is not match L532

**Review by angelo-chan — APPROVED** [review]

**Pull request merged**